File: SeekMatch-Client/src/app/shared/models/company.ts
export class Company {
  
  public name: string = '';
  public phone: string = '';
  public address: string = '';
  public country: string = '';
  public region: string | null = null;
  public city: string | null = null;
  public description: string = '';
  public website: string = '';

  constructor(init?:Partial<Company>) {
    Object.assign(this, init);
  }
}

File: SeekMatch-Server/SeekMatch.Core/Entities/Representative/Company.cs
namespace SeekMatch.Core.Entities
{
    public class Company : BaseEntity
    {
        public required string Name { get; set; }
        public required string Phone { get; set; }
        public required string Address { get; set; }
        public string? Country { get; set; }
        public int? Region { get; set; }
        public int? City { get; set; }
        public string? Description { get; set; }
        public string? Website { get; set; }
        public List<Representative> Representatives { get; set; } = new List<Representative>();
        public List<Recruiter> Recruiters { get; set; } = new List<Recruiter>();
    }
}
